core/templates/components/summary-tile/story-summary-tile.directive.ts

```diff
@@ -41,6 +41,9 @@ angular.module('oppia').directive('storySummaryTile', [
             ENTITY_TYPE, STORY_VIEWER_URL_TEMPLATE) {
           var ctrl = this;
           ctrl.getStoryLink = function() {
+            if (!ctrl.classroomUrlFragment || !ctrl.topicUrlFragment) {
```

Comment:
Can we add a comment as to why we are doing this?

Reply:
Added.